proto/src/main/proto/Rss.proto

```diff
@@ -321,3 +324,13 @@ message CheckServiceAvailableResponse {
   StatusCode status = 1;
   bool available = 2;
 }
+
+message AccessClusterRequest {
+  StatusCode status = 1;
```

Comment:
why do we need status?

Reply:
The client needs to know whether it can access and use this information to decide to use rss shuffle or native shuffle.

Reply:
This is request, not response.

Reply:
My fault, will fix it.